<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenlCamCap</class>
 <widget class="QWidget" name="GenlCamCap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Control Panel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="DiviceControl">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>246</width>
        <height>267</height>
       </rect>
      </property>
      <attribute name="label">
       <string>DiviceControl</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3"/>
     </widget>
     <widget class="QWidget" name="ImageFormatControl">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>246</width>
        <height>267</height>
       </rect>
      </property>
      <attribute name="label">
       <string>ImageFormatControl</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QFormLayout" name="ImageFormatControlForm">
         <item row="0" column="0">
          <widget class="QLabel" name="widthLabel">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="widthSpinBox">
           <property name="maximum">
            <number>2592</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="heightLabel">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="heightSpinbox">
           <property name="maximum">
            <number>2048</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AcquisitionControl">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>246</width>
        <height>267</height>
       </rect>
      </property>
      <attribute name="label">
       <string>AcquisitionControl</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="frameRateEnableLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>FrameRateEnable</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="frameRateEnableCombobox">
           <item>
            <property name="text">
             <string>True</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>False</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="frameRateLabel">
           <property name="text">
            <string>FrameRate</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="frameRateSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>60.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="exposureAutoLabel">
           <property name="text">
            <string>ExposureAuto</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="exposureAutoCombobox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="exposureTimeLabel">
           <property name="text">
            <string>ExposureTime</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="exposureTimeSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ISPControl">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>246</width>
        <height>267</height>
       </rect>
      </property>
      <attribute name="label">
       <string>ISPControl</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QFormLayout" name="ISPControlForm">
         <item row="0" column="0">
          <widget class="QLabel" name="brightnessLabel">
           <property name="text">
            <string>Brightness</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="brightnessSpinBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="sharpnessLabel">
           <property name="text">
            <string>Sharpness</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="sharpnessSpinBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="digitalShiftLabel">
           <property name="text">
            <string>DigitalShift</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="digitalShiftSpinBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="UserSetControl">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>246</width>
        <height>267</height>
       </rect>
      </property>
      <attribute name="label">
       <string>UserSetControl</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
